gammapy/data/tests/test_observation_summary.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+from astropy.coordinates import Angle, SkyCoord
+
+from gammapy.data import DataStore
+from gammapy.data import ObservationTable
+from gammapy.data import ObservationTableSummary
```

Comment:
In Gammapy, we use explicit relative imports everywhere.
So to be consistent, please change this to
```python
Remove this unused import:
```python
from ...data import DataStore, ObservationTable, ObservationTableSummary
```